src/server/scripts/Northrend/IcecrownCitadel/icecrown_citadel.cpp

```diff
@@ -1043,6 +1055,80 @@ struct npc_darkfallen_tactician : public DarkFallenAI
     }
 };
 
+struct npc_nerubar_broodkeeper : public ScriptedAI
+{
+    npc_nerubar_broodkeeper(Creature* creature) : ScriptedAI(creature)
+    {
+        Initialize();
+    }
+
+    void Initialize()
+    {
+        CastNet = false;
+    }
+
+    void Reset() override
+    {
+        events.Reset();
```

Comment:
shouldn't Reset() set CastNet to false too (which is the whole point of having Initialize() ) ?

Reply:
> shouldn't Reset() set CastNet to false too (which is the whole point of having Initialize() ) ?

I don't understand much here, I don't speak English very well. I didn't declare Initialize (); in Reset () so?

Reply:
let me try again:
Shouldn't CastNet set to false again on Reset() ? or is there a reason for that not to happen ?

Reply:
And exactly, if it goes to the standard position, CastNet will not work)